lib/broadside/target.rb

```diff
@@ -74,14 +85,29 @@ def family
       "#{Broadside.config.application}_#{@name}"
     end
 
-    def to_h
+    def to_hash
```

Comment:
i don't feel strongly about this _but_ the amazon classes all implement `to_hash`; that's the only reason i went with it in broadside.  but i'll change it if you want.